app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -540,7 +541,11 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
         val intent = Intent(appContext,toClass)
         return TaskStackBuilder.create(appContext).run {
              addNextIntentWithParentStack(intent)
-             getPendingIntent(0, PendingIntent.FLAG_UPDATE_CURRENT)
+             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                 getPendingIntent(0, PendingIntent.FLAG_IMMUTABLE)
```

Comment:
IIUC, I believe we should be keeping both the `FLAG_IMMUTABLE` and `FLAG_UPDATE_CURRENT` flags [ [ref](https://developer.android.com/reference/android/app/PendingIntent#FLAG_UPDATE_CURRENT) ]. Is there any reason behind not sending the `FLAG_UPDATE_CURRENT` here? Have you explored on what difference including / not including it makes ?